### PR TITLE
refactor: Checking param convention and removing redundant code

### PR DESCRIPTION
**Pull request by Dylan-Prins**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Dylan-Prins** [comment]
Depended on #42 which contains fix for tests

File: AzureDevOpsPowerShell/Public/Api/Graph/Groups/Set-AzDoTeamMember.ps1 [hunk 1]
```diff
@@ -41,26 +41,19 @@ function Set-AzDoTeamMember {
...
+    Write-Verbose "Starting function: Set-AzDoTeamMember"
   }
 
   process {
     $params = @{
       uri     = "https://vssps.dev.azure.com/$OrganizationName/_apis/graph/groups?api-version=7.1-preview.1"
-      Method  = 'GET'
-      Headers = @{Authorization = 'Basic ' + [Convert]::ToBase64String([Text.Encoding]::ASCII.GetBytes(":$($PAT)")) }
+      method  = 'GET'
+      headers = @{Authorization = 'Basic ' + [Convert]::ToBase64String([Text.Encoding]::ASCII.GetBytes(":$($PAT)")) }
```
Comment (Manbearpiet) [inline AzureDevOpsPowerShell/Public/Api/Graph/Groups/Set-AzDoTeamMember.ps1:51]: Do we want to retain the function in this state, I'd delete it.

File: AzureDevOpsPowerShell/Public/Api/Policy/Configuration/Get-AzDoBranchPolicy.ps1 [hunk 2]
```diff
@@ -46,23 +46,10 @@ function Get-AzDoBranchPolicy {
...
-      } else {
-        $result.add($policies) | Out-Null
-      }
+      $result += (Invoke-AzDoRestMethod @params).value | Where-Object { -not $PolicyName -or $_.Name -in $PolicyName }
 
     } else {
-      $body | Format-List
+      Write-Verbose "Calling Invoke-AzDoRestMethod with $($params| ConvertTo-Json -Depth 10)"
     }
   }
```
Comment (Manbearpiet) [inline AzureDevOpsPowerShell/Public/Api/Policy/Configuration/Get-AzDoBranchPolicy.ps1:55]: Flat output of $result?

**Review by Manbearpiet — COMMENTED** [review]
Added comments

**Review by Manbearpiet — APPROVED** [review]
LGTM

**Pull request merged**